src/UI/ValueFactory.cs

```diff
@@ -388,5 +406,33 @@ private static bool GetNewColorSchemeValue(Design design, out object? newValue)
                 return false;
             }
         }
+
+        /// <summary>
+        /// Returns true if path is to a directory.
+        /// Thanks: https://stackoverflow.com/a/19596821/4824531
+        /// </summary>
+        /// <param name="path"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        static bool IsPathDirectory(string path)
+        {
+            if (path == null) throw new ArgumentNullException("path");
```

Comment:
Directory.Exists already checks for null and trims ending whitespace on Windows (conditionally).

We shouldn't do it ourselves, because whitespace is perfectly valid on linux:

```
:~/dev/test$ mkdir \ \ This\ Begins,\ Contains,\ And\ Ends\ In\ Whitespace\ \ \ \
:~/dev/test$ ls -1
Program.cs
test.csproj
'  This Begins, Contains, And Ends In Whitespace    '
```

Reply:
Example output from a program written to output Directory.Exists for the specified path on linux:

```
:~/dev/test$ for i in . .. ./ ../ ~ ~/ ./\ \ This\ Begins\,\ Contains\,\ And\ Ends\ In\ Whitespace\ \ \ \  ; do dotnet run "${i}" ; done
Does . exist? yes
Does .. exist? yes
Does ./ exist? yes
Does ../ exist? yes
Does /home/devuser exist? yes
Does /home/devuser/ exist? yes
Does ./  This Begins, Contains, And Ends In Whitespace     exist? yes
```